AK/Platform.h

```diff
@@ -166,7 +166,7 @@ extern "C" {
 #    define CLOCK_MONOTONIC_COARSE CLOCK_MONOTONIC
 #endif
 
-#if defined(AK_OS_BSD_GENERIC)
+#if defined(AK_OS_BSD_GENERIC) && !defined(__FreeBSD__)
```

Comment:
This change seems fine, but could you change it to `!defined(AK_OS_FREEBSD)` ? We recently added some fine grained macros and got rid of as many of the `__platform__` macros as possible in the codebase

Reply:
i have pushed up an update to use ```!defined(AK_OS_FREEBSD)``` instead of the platform specific one. thank you for catching that, thank you!